File: src/main/java/com/menighin/luwak/core/dtos/LuwakPageMetadataDto.java
package com.menighin.luwak.core.dtos;

import lombok.Getter;
import lombok.Setter;

import java.util.ArrayList;

public class LuwakPageMetadataDto {

	@Getter @Setter
	private String title;

	@Getter @Setter
	private LuwakDataTableMetadataDto masterTable;

	@Getter @Setter
	private LuwakDataTableMetadataDto slaveTable;

	@Getter @Setter
	private ArrayList<LuwakFilterMetadataDto> filters;

	public LuwakPageMetadataDto() {
		filters = new ArrayList<>();
	}

}

File: src/main/java/com/menighin/luwak/core/dtos/LuwakPageMetadataDto.kt
package com.menighin.luwak.core.dtos

import java.util.ArrayList

class LuwakPageMetadataDto {

	var title: String? = null

	var masterTable: LuwakDataTableMetadataDto? = null

	var detailTables: List<LuwakDataTableMetadataDto>? = null

	var filters: ArrayList<LuwakFilterMetadataDto> = ArrayList()

}
